variables.tf

```
@@ -34,10 +34,22 @@ variable "dry_run" {
 }
 
 variable "lambda" {
-  description = "Object of attributes for the lambda function configuration"
+  description = "Object of optional attributes passed on to the lambda module"
   type = object({
-    runtime = optional(string, "python3.9")
+    artifacts_dir            = optional(string, "builds")
+    build_in_docker          = optional(bool, false)
+    create_package           = optional(bool, true)
+    ephemeral_storage_size   = optional(number)
+    ignore_source_code_hash  = optional(bool, true)
+    local_existing_package   = optional(string)
+    memory_size              = optional(number, 128)
+    recreate_missing_package = optional(bool, false)
+    runtime                  = optional(string, "python3.8")
+    s3_bucket                = optional(string)
+    s3_existing_package      = optional(map(string))
+    s3_prefix                = optional(string)
+    store_on_s3              = optional(bool, false)
+    timeout                  = optional(number, 300)
   })
-  nullable = false
```

We can leave this, it's probably a good idea to set it on this particular variable...

```suggestion
  nullable = false
```